tests/publiccloud/run_ltp.pm

```diff
@@ -45,35 +45,41 @@ sub instance_log_args
 sub run {
     my ($self)   = @_;
     my $arch     = check_var('PUBLIC_CLOUD_ARCH', 'arm64') ? 'aarch64' : 'x86_64';
-    my $ltp_repo = get_var('LTP_REPO', 'http://download.suse.de/ibs/QA:/Head/' . generate_version("-") . '/' . $arch . '/');
+    my $ltp_repo = get_var('LTP_REPO', 'https://download.opensuse.org/repositories/benchmark:/ltp:/devel/' . generate_version("-") . '/');
     my $REG_CODE = get_required_var('SCC_REGCODE');
```

Comment:
@asmorodskyi Have you tested this? You're using `-` as separator, but IMHO you need for SLE the default `_` as separator (second `_` in `SLE_15_SP3`), because `VERSION` e.g. `15-SP3`.

Reply:
you right , will change this